README.md

```diff
@@ -39,7 +39,7 @@ NB: the default batchSize is much higher than the throughput the instance data i
 ## Endpoints
 /topics/{uuid}
 ### PUT
-The only mandatory field is the uuid, and the uuid in the body must match the one used on the path.
+The only mandatory field is the uuid and prefLabel and alternativeIdentifiers.uuids. The uuid in the body must match the one used on the path.
```

Comment:
"The only mandatory fields are the uuid, the prefLabel and the alternativeIdentifier uuids (because the uuid is also listed in the alternativeIdentifier uuids list)."

Suggest a slight rewording for clarity.